kodilla-stream/src/main/java/com/kodilla/stream/StreamMain.java

```diff
@@ -13,5 +18,17 @@ public static void main(String[] args) {
                 .collect(Collectors.joining(",\n","<<",">>"));
 
         System.out.println(theResultStringOfBooks);
+
+        Forum forum = new Forum();
+
+        Map<Integer, ForumUser> theResultMapOfUsers = forum.getUserList().stream()
+                .filter(forumUser -> forumUser.getSex() == 'M')
+                .filter(forumUser -> forumUser.getBirthDate().isBefore(LocalDate.now().minusYears(20)))
+                .filter(forumUser -> forumUser.getPostsNumber() >= 1)
+                .collect(Collectors.toMap(ForumUser::getUserId, forumUser -> forumUser));
+
+        theResultMapOfUsers.entrySet().stream()
```

Comment:
Zbyt somplikowana nazwa `theResultMapOfUsers` - proponuję prościej jakoś (np. '`filteredUsers') - nie zdradzamy, że mamy mapę (to widać)
Oczywiście w celu przećwiczenia streamów jest to okej, aczkolwiek normalnie w pracy takie rzeczy możemy łatwiej zrobić a pomocą biblioteki (bardzo pomocnej w wielu kwestiach) `Guava`. 
Wówczas kod wygląda tak: `System.out.println(Joiner.on("\n").withKeyValueSeparator(":").join(theResultMapOfUsers));` i mamy wydrukowaną mapę.
Maven dependency:
<dependency>
    <groupId>com.google.guava</groupId>
    <artifactId>guava</artifactId>
    <version>27.1-jre</version>
</dependency>